The check for Backtrace_FOUND must use #if not #ifdef,
because there is a "#define Backtrace_FOUND 0" in case
there is no backtrace() functionality (i.e. musl libc)

--- shared/akcrash.cpp	2014-08-10 12:38:58.000000000 +0200
+++ shared/akcrash.cpp	2015-07-28 14:32:27.230877108 +0200
@@ -39,7 +39,7 @@
   QString s;
 
 /* FIXME: is there an equivalent for windows? */
-#ifdef Backtrace_FOUND
+#if Backtrace_FOUND
   void *trace[256];
   int n = backtrace( trace, 256 );
   if ( !n ) {
